standalone/standalone.py

```diff
@@ -311,7 +311,7 @@ def upload_s3_file():
 
 if [ "$STRATEGY" == "download" ]; then
   # List top-level directories only (no nested directories)
-  top_level_dirs=$(tar --exclude='*/*' --list --file {data_pvc_mount_path}/data.tar.gz)
+  top_level_dirs=$(tar  -tzf {data_pvc_mount_path}/data.tar.gz | grep '/$')
```

Comment:
I'm actually removing this part in another PR (will push soon). This takes as long as untarring so not ideal... Can you remove this? Thanks!

Reply:
removed!